nebula/ui/components/VPNFlickable.qml

```diff
@@ -64,7 +64,28 @@ Flickable {
         ensureVisAnimation.start();
     }
 
-    contentHeight: Math.max(window.safeContentHeight, flickContentHeight)
+    function recalculateContentHeight() {
+        //Checks if the flickable AND it's content interferes with the navbar area (bottom 146px for iOS, bottom 128px for other platforms)
+        //If it does, we pad the flickable's contentHeight with whatever bottom padding is needed so that there is always 48px (aka theme.navBarTopMargin)
+        //between the bottom of the flickable's content and the navbar
+        if (navbar.visible && mapToItem(window.contentItem, 0, 0).y + height >= window.height - VPNTheme.theme.navBarHeightWithMargins
+                && flickContentHeight + mapToItem(window.contentItem, 0, 0).y >= window.height - VPNTheme.theme.navBarHeightWithMargins) {
```

Comment:
Nit: What do you think of assigning `mapToItem(window.contentItem, 0, 0).y` and `window.height - VPNTheme.theme.navBarHeightWithMargins` to a property with a descriptive name for better readability and reduced redundancy? They are used a couple of times in this file.

Reply:
Good idea!